### PR TITLE
migrate all scalability envs into presets

### PR DESCRIPTION
**Pull request by krzyzacy**
/hold

we probably need to make presets able to overwrite env vars respect the order of presets are set :-/ I'll see if I can find a workaround.

cc @shyamjvs @wojtek-t

### CODE REVIEW

File: config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml [hunk 1]
```diff
@@ -526,8 +527,8 @@ periodics:
       - --scenario=kubernetes_e2e
       - --
       - --cluster=kubemark-100pods
-      - --env-file=jobs/env/ci-kubernetes-e2e-kubemark-common.env
-      - --env-file=jobs/env/ci-kubernetes-kubemark-high-density-100-gce.env
+      # Reduce logs verbosity
+      - --env=TEST_CLUSTER_LOG_LEVEL=--v=2
```
Comment (wojtek-t) [inline config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml]: Can you add a TODO for me to get rid of that (or unify with other jobs where we reduce verbosity)?

Comment (krzyzacy) [reply config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml]: if it's not needed I can get rid of it? (Looks like it only references kubemark-common which doesn't have the verbosity option)

Comment (wojtek-t) [reply config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml]: Log verbosity has impact on performance. So let's not mix such changes with biggest refactorings.
Please add a TODO for me to validate if we can safely do that.

File: config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml [hunk 2]
```diff
@@ -46,8 +47,9 @@ periodics:
       - --scenario=kubernetes_e2e
       - --
       - --cluster=e2e-big-beta
-      - --env-file=jobs/env/ci-kubernetes-e2e-scalability-common.env
-      - --env-file=jobs/env/ci-kubernetes-e2e-scalability-old-common.env
+      # Use default kube-qps and kube-api-burst flag values.
+      - --env=CONTROLLER_MANAGER_TEST_ARGS=--profiling
+      - --env=SCHEDULER_TEST_ARGS=--profiling
```
Comment (krzyzacy) [inline config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml]: I think it's ok to explicitly set these two, as when we advance k8s versions we can get rid of them per release-branch.

**Comment by wojtek-t** [comment]
@krzyzacy - can we not merge this on Friday and wait until Monday? [just in case]

**Comment by krzyzacy** [comment]
@wojtek-t sure, I'll leave you to `/hold cancel`

**Pull request merged**